### DISCUSSION TITLE
How to view the contents of a rofl file

### DISCUSSION THREAD

**Discussion by trhao99** [fraxiinus/ReplayBook#183] [General]
May I ask what encoding format should I use to open the rofl file to view the content in it? I use utf-8 to open the file, but I see garbled characters

**Comment by fraxiinus** [comment]
You can look at one of the implementations here: https://github.com/fraxiinus/roflxd

or you can read the ROFL specification docs: https://github.com/robertabcd/lol-ob/wiki/ROFL-Container-Notes
